core/src/main/scala/org/genivi/sota/core/rvi/TransferController.scala

```diff
@@ -180,7 +178,9 @@ class TransferProtocolActor(db: Database, rviClient: RviClient,
             db.run(OperationResults.persist(org.genivi.sota.core.data.OperationResult(
```

Comment:
as far as I understand it, a successful install based on some update spec must originate from a persisted update request, so logging the install history based on the update_request_id should be ok

but yea your gist looks cleaner and should be equivalent